web/public/js/directives/cd-start-dojo/venue/template.dust

```diff
@@ -130,6 +101,44 @@
             ng-model="$ctrl.venue.alternativeCorporate"></input>
         </div>
       </div>
+      <div class="cd-simple-flex-grid cd-sad-venue__online">
+        <div class="cd-simple-flex-grid__flex-1 cd-sad-venue__online">
+          <div class="checkbox-inline">
+            <input type="checkbox" id="online_dojo" ng-model="$ctrl.venue.onlineSessions"
+              name="onlineSessions"
+              ng-false-value="0"
+              ng-true-value="1"
+              ng-checked="$ctrl.venue.onlineSessions == 1">
+            <label for="online_dojo">{@i18n key="This Dojo will run online (partially or completely)"/}</input>
+          </div>
+          <p class="help-block">{@i18n key="Check this box if your club runs online all or some of the time."/}</p>
+        </div>
+      </div>
+      <div class="cd-simple-flex-grid cd-sad-venue__safeguarding">
+        <div class="cd-simple-flex-grid__flex-1 cd-sad-venue__safeguarding">
+          <h3>{@i18n key="Keeping your Dojo safe"/}</h3>
+          <p>
+            {@i18n key="To ensure the safety of young people at Dojos, we require the email address of a safeguarding sponsor."/} &nbsp;
+            {@i18n key="An appropriate sponsor would be a person holding a manager-level position within a corporate or non-profit organisation."/} &nbsp;
+            {@i18n key="This sponsor will help ensure that your club operates in accordance with our safeguarding policy and that all your volunteers have the appropriate background checks."/}
```

Comment:
Nuala suggested that *our safeguarding policy* should be linked to https://www.raspberrypi.org/safeguarding/ which makes a lot of sense to me, apart from that she was happy with the changes.

Reply:
Done

![Screenshot 2020-11-06 at 11 38 27](https://user-images.githubusercontent.com/789846/98366322-b195d000-202b-11eb-9982-f9643db2c105.png)